runtime/moonbeam/src/lib.rs

```diff
@@ -1318,6 +1318,24 @@ impl pallet_randomness::Config for Runtime {
 
 impl pallet_root_testing::Config for Runtime {}
 
+parameter_types! {
+	// One storage item; key size is 32; value is size 4+4+16+20 bytes = 44 bytes.
+	pub const DepositBase: Balance = currency::deposit(1, 76);
+	// Additional storage item size of 32 bytes.
+	pub const DepositFactor: Balance = currency::deposit(0, 20);
```

Comment:
32 or 20?

Reply:
Not sure what you meant, but I agree that the doc saying 32 and the code saying 20 is confusing

Reply:
I don't think so @librelois, because it is "The amount of currency needed per unit threshold when creating a multisig execution", it doesn't take into account the size of AccountId like `DepositBase`. Here are their definitions in `pallet-multisig`.

```rust
/// The base amount of currency needed to reserve for creating a multisig execution or to
/// store a dispatch call for later.
///
/// This is held for an additional storage item whose value size is
/// `4 + sizeof((BlockNumber, Balance, AccountId))` bytes and whose key size is
/// `32 + sizeof(AccountId)` bytes.
#[pallet::constant]
type DepositBase: Get<BalanceOf<Self>>;

/// The amount of currency needed per unit threshold when creating a multisig execution.
///
/// This is held for adding 32 bytes more into a pre-existing storage value.
#[pallet::constant]
type DepositFactor: Get<BalanceOf<Self>>;
```

Reply:
@Agusrodri you just looked at their documentation, I I looked at their code, and their documentation is wrogn, it's only true for chains with 32 bytes based accounts.

 The goal of the DepositFactor is to account for the exact storage size that is needed Looks at their storages item structure, you will see that the factor variation only affect AccountIds

Reply:
@librelois okay, I just changed it